src/common/plugin.ts

```diff
@@ -45,12 +48,31 @@ export abstract class Plugin implements IPlugin {
     const channelName = channel.name;
     const categoryName = channel.parent?.name.toLowerCase();
 
-    if (this.pluginChannelName && this.pluginChannelName !== channelName) {
-      const id = this.container.guildService.getChannel(this.pluginChannelName).id;
-      return `Please use this command in the <#${id}> channel.`;
+    let allowedChannels: string[] | undefined;
+    if (this.pluginChannelName) {
```

Comment:
This is much better. I still think it can be a bit cleaner than all these cases though

Reply:
Cleaned up more. Plugin developers can now provide allowed channels either using the single string field, or any number of strings in the array field.